src/main/resources/META-INF/rewrite/jakarta-ee-10.yml

```diff
@@ -308,4 +309,30 @@ recipeList:
   - org.openrewrite.java.ChangeType:
       oldFullyQualifiedTypeName: jakarta.faces.bean.ViewScoped
       newFullyQualifiedTypeName: jakarta.faces.view.ViewScoped
-      ignoreDefinition: true
\ No newline at end of file
+      ignoreDefinition: true
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.migrate.jakarta.JavaxBeansXmlToJakartaBeansXml
+displayName: Migrate xmlns entries in `beans.xml` files
+description: Java EE has been rebranded to Jakarta EE, necessitating an XML namespace relocation.
+tags:
+  - jakarta
+  - beans
+  - cdi
+recipeList:
+  - org.openrewrite.xml.ChangeTagAttribute:
+      attributeName: version
+      elementName: beans
+      newValue: 3.0
+  - org.openrewrite.xml.ChangeTagAttribute:
+      attributeName: xmlns
+      elementName: beans
+      newValue: https://jakarta.ee/xml/ns/jakartaee
+  - org.openrewrite.xml.ChangeTagAttribute:
+      attributeName: xsi:schemaLocation
+      elementName: beans
+      newValue: https://jakarta.ee/xml/ns/jakartaee https://jakarta.ee/xml/ns/jakartaee/beans_3_0.xsd
```

Comment:
I'm wondering if it would make sense to work this into what I think is a related recipe; any thoughts there @melloware & @ranuradh ?

https://github.com/openrewrite/rewrite-migrate-java/blob/8733da70af9f54faf73f7b7f5b6d23d089b1cb6d/src/main/java/org/openrewrite/java/migrate/BeansXmlNamespace.java#L32-L70

Reply:
I was wondering why this legacy beans.xml was done this way?